R/constants.R

```diff
@@ -335,7 +338,11 @@ CONV_HA_M2 <- 1e4 # ha to m2
 CONV_BM2_M2 <- 1e9
 CONV_MILFT2_M2 <- 92900 # Million square feet to square meters
 CONV_FT2_M2 <- 0.0929 # Square feet to square meters
+CONV_C_CO2 <- 44/12 # come from _common/assumptions/units_conversions.R
```

Comment:
Doesn't this already occur elsewhere is `constants.R`?

Reply:
Yes and it actually got defined twice under emissions section. Duplicates removed.